resources/views/alert.blade.php

```diff
@@ -3,17 +3,18 @@
     'messageClass' => 'text-sm',
     'type' => 'info',
     'large' => false,
+    'rounded' => false,
```

Comment:
is this in the designs that we need a rounded icon version?

Reply:
Went through the designs and nope, not for MSQ at least, worth checking elsewhere or I just trash the whole rounded ? 🤔

Reply:
ask oleg

Reply:
Yeah we ditch the round icons for alerts, so removing this.